admin/includes/modules/dashboard_widgets/OrderStatusDashboardWidget.php

```diff
@@ -21,7 +21,7 @@
           $orders_pending = $db->Execute("SELECT count(*) as count FROM " . TABLE_ORDERS . " WHERE orders_status = " . (int)$row['orders_status_id'], false, true, 1800);
           ?>
         <tr>
-          <td><a href="<?php echo zen_href_link(FILENAME_ORDERS, 'selected_box=customers&statusFilterSelect=' . $row['orders_status_id']); ?>"><?php echo $row['orders_status_name']; ?></a>:</td>
+          <td><a href="<?php echo zen_href_link(FILENAME_ORDERS, 'customers&statusFilterSelect=' . $row['orders_status_id']); ?>"><?php echo $row['orders_status_name']; ?></a>:</td>
```

Comment:
Shouldn't the `customers&` portion be removed as well?

Reply:
You are right, I'll update the PR